src/main/thrift/parquet.thrift

```diff
@@ -914,6 +1040,32 @@ struct RowGroup {
   7: optional i16 ordinal
 }
 
+struct RowGroupV3 {
+  /** REMOVED from v1: columns.
+   * Instead, decode each FileColumnMetadataV3 individually as needed.
+   */
+
+  /** Total byte size of all the uncompressed column data in this row group **/
+  1: required i64 total_byte_size
```

Comment:
I'm not sure if this field is particularly useful, as it doesn't account for encodings.  We might want a different field name with a clearer meaning.

Reply:
This field was introduced [in 2013](https://github.com/apache/parquet-format/commit/e52693de173387d95300cd3e3ea67a04aa81d789), I'm not sure anyone still has a memory of the reasons. The author is named Nong Li. @nongli Is it you?
Also ping @julienledem, who was there too.

Reply:
I guess it might be used for a proxy as to actual data size, but it isn't a good one maybe we can keep it and always add another better field.

Reply:
I'm ok with removing this, FTR, I would just like to make sure this isn't actually useful for some unexpected use case.

Reply:
I believe it was added before that. this commit is just referencing when we created the separate parquet-format repo. It might well be part of the original metadata. The `redfile.thrift` name refers to the old RedElm name before we rename it Parquet. It is the size of the decompressed (after decompression, before decoding) page. Agreed it's not super useful.